README.md

```diff
@@ -1 +1,106 @@
-# vacations-booking-back-end
+![](https://img.shields.io/badge/microverse-blueviolet)
+
+# Vacations booking back-end
+
+> "Vacations booking back-end" project is a Rails API application which provide the API endpoints for creating/removing different destinations by admin users and creating reservations by other logged users.
+
+![image](https://user-images.githubusercontent.com/63932912/143224920-5cc44446-f6f1-47ee-b737-78e24e5613f7.png)
+
+
+## Built With
+
+- Ruby on Rails
+- Postgres
+
+
+## Getting Started
+
+To get a local copy up and run the app, follow these simple example steps.
+
+### Prerequisites
+
+- Ruby > 2.7.3
+- Rails > 6.1.4.1
+- Postgres > 14.0
+
+### Setup
+
+Clone the repository with:
+
+```
+git clone git@github.com:mugberto/vacations-booking-back-end.git
+```
+or download [ZIP file](https://github.com/mugberto/vacations-booking-back-end/archive/refs/heads/dev.zip)
+
+### Install
+To install all dependencies, run:
+```
+bundle install
+```
+Create database and seed the data
+```
+rails db:create
+rails db:migrate
+rails db:seed
+```
+### Usage
+To run the server, in the project directory, run:
+
+ ```
+ rails s
+ ```
+ 
```

Comment:
This is good

[OPTIONAL] - See my comment on your front-end PR 51-  having a workflow for testing to make it easier for team members to see changes live